src/opower/opower.py

```diff
@@ -17,7 +17,7 @@
 from .utilities import UtilityBase
 
 _LOGGER = logging.getLogger(__file__)
-DEBUG_LOG_RESPONSE = False
+DEBUG_LOG_RESPONSE = True

```

Comment:
Reminder for you to revert this line. I'd like to introduce a very_verbose flag or similar for this. If you could separately do it yourself it would be great :)

Reply:
Yep, it was a temporary solution just to help debug the issues. I also thought that we could enable this option by passing the -vv key :)
Removed for now